### DISCUSSION TITLE
How to get environment variables value?

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [c4spar/cliffy#532] [Q&A] [answered]
I need requirements to get the value of the deno global environment variable. The environment variable name is `DENO_INSTALL_ROOT`. I can not get the environment variable value. I use `deno.env.get("DENO_INSTALL_ROOT")`. But it does not work for me.

**Comment by c4spar** [comment, marked as answer]
This is not related to cliffy. If `Deno.env.get("DENO_INSTALL_ROOT")` returns undefined than the env var is not set. But if you have issues with deno you should open an issue/discussion in the deno repository.

If you are looking for a way how to use environment variables with cliffy you can take a look at https://cliffy.io/docs@v0.25.6/command/environment-variables.

**Reply by officialrajdeepsingh** [reply]
I `Deno.env.get("DENO_INSTALL_ROOT") `  try it. But it does not work for me.
